src/controller/java/AndroidCallbacks.cpp

```diff
@@ -448,6 +448,10 @@ void ReportEventCallback::OnEventData(const app::EventHeader & aEventHeader, TLV
     readerForJavaTLV.Init(*apData);
     readerForJson.Init(*apData);
 
+    jlong eventNumber   = static_cast<jlong>(aEventHeader.mEventNumber);
+    jlong priorityLevel = static_cast<jint>(aEventHeader.mPriorityLevel);
+    jlong timestamp     = static_cast<jlong>(aEventHeader.mTimestamp.mValue);
+
```

Comment:
why don't you add timestamp type? per spec, it could be either epoch or system timestamp.

Reply:
https://github.com/project-chip/connectedhomeip/issues/20595